ledger/participant-integration-api/src/main/scala/platform/apiserver/services/admin/ApiUserManagementService.scala

```diff
@@ -82,14 +84,15 @@ private[apiserver] final class ApiUserManagementService(
         .map(_ => proto.DeleteUserResponse())
     )
 
-  override def listUsers(request: proto.ListUsersRequest): Future[proto.ListUsersResponse] =
+  override def listUsers(request: proto.ListUsersRequest): Future[proto.ListUsersResponse] = {
     userManagementService
-      .listUsers()
+      .listUsers(pageToken = request.pageToken, maxResults = request.pageSize)
```

Comment:
TODO: add a configurable max page size limit to the server itself to defend against clients requesting huge results.

Reply:
Ok.
Will expose it in `UserManagementFeature` as well